src/main/resources/perforce/ReleaseTrigger.py

```diff
@@ -6,32 +6,43 @@
 
 import sys
 from com.perforce.p4java.server import ServerFactory
+from com.perforce.p4java.option.server import GetChangelistsOptions
+from com.perforce.p4java.core import IChangelist
+from com.perforce.p4java.core.file import FileSpecBuilder
+from com.perforce.p4java import PropertyDefs
+from java.lang import System
 
 if perforceServer is None:
-   print "No server provided"
    sys.exit(1)
 
-iServer = ServerFactory().getServer(perforceServer['url'], None)
+props = System.getProperties()
+props.put(PropertyDefs.PROG_NAME_KEY, "P4-XL_Release")
+props.put(PropertyDefs.PROG_VERSION_KEY, "2.0.0")
 
-if username:
-   iServer.setUserName(userName)
-else:
-   iServer.setUserName(perforceServer['username'])
+iServer = ServerFactory().getOptionsServer(perforceServer['port'], props)
```

Comment:
Would "uri" or "url" be better than "port" here, since the script expects more than just a port number?  A change to the name also affect synthetic.xml and image #1 above.